packages/netlify-cms-backend-bitbucket/src/implementation.ts

```diff
@@ -373,26 +373,26 @@ export default class BitbucketBackend implements Implementation {
     return [];
   }
 
-  async unpublishedEntry(collection: Collection, slug: string) {
+  async unpublishedEntry(collection: Collection, _slug: string) {
     if (collection) {
       throw new EditorialWorkflowError('content is not under editorial workflow', true);
     }
     return { data: '', file: { path: '' } };
   }
 
-  async updateUnpublishedEntryStatus(collection: string, slug: string, newStatus: string) {
+  async updateUnpublishedEntryStatus(_collection: string, _slug: string, _newStatus: string) {
     return;
   }
 
-  async publishUnpublishedEntry(collection: string, slug: string) {
+  async publishUnpublishedEntry(_collection: string, _slug: string) {
     return;
   }
 
-  async deleteUnpublishedEntry(collection: string, slug: string) {
+  async deleteUnpublishedEntry(_collection: string, _slug: string) {
     return;
   }
 
-  async getDeployPreview(collectionName: string, slug: string) {
+  async getDeployPreview(_collection: string, _slug: string) {
```

Comment:
Are the underscores meant to denote that these params are unused?

Reply:
Exactly. I added `noUnusedParameters: true` to `tsconfig.json` after adding TypeScript to the backends to identify methods with redundant arguments to make it easier to refactor.
Initially added the underscores to the methods I haven't implemented yet, but knew I'm going to use the arguments later...